index.html

```diff
@@ -1,893 +1,952 @@
 <!doctype html>
 <html lang="en-US">
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+<head>
```

Comment:
Please don't autoformat the existing lines.

Reply:
Sorry, fixed it.